libs/eslint-plugin-vx/src/configs/recommended.ts

```diff
@@ -175,12 +175,7 @@ export = {
     },
     {
       files: ['**/*.test.ts', '**/*.test.tsx'],
-      plugins: ['jest'],
       rules: {
-        'jest/max-nested-describe': ['error', { max: 1 }],
-        'jest/no-identical-title': 'error',
-        'jest/no-focused-tests': 'error',
-        'jest/valid-expect': ['error', { alwaysAwait: true }],
```

Comment:
Do we want any similar lint rules for `vitest`?

Reply:
Working on adding `eslint-plugin-vitest`.

Reply:
[8b93ef6](https://github.com/votingworks/vxsuite/pull/6032/commits/8b93ef6b763fe2271e27aba555d9412022f14598)